src/devices/src/virtio/vsock/device.rs

```diff
@@ -75,6 +76,7 @@ where
     B: VsockBackend,
 {
     pub(crate) fn with_queues(
+        id: String,
```

Comment:
That's right, I didn't want to introduce API breaking changes with this PR so it stays focused.

A `String` `id` is used in our device map which is hashed based on `(DeviceType, id)` so we can't just not use one at all for `Vsock`.

I will make a change to this PR to simply ignore the one from API (it's not useful since we only ever have a single vsock), and hardcode a vsock id string in the vsock implementation. A `vsock.id()` will just provide a static hardcoded id.

What do you think?